mithril.js

```diff
@@ -141,11 +141,9 @@
 	 *                      or splat (optional)
 	 */
 	function m(tag, pairs) {
-		var args = []
-
-		for (var i = 1, length = arguments.length; i < length; i++) {
-			args[i - 1] = arguments[i]
-		}
+		var args = Array.prototype.slice.call(arguments, 1).map(function(x) {
```

Comment:
`var args = Array.prototype.slice.call(arguments, 1)` is enough, no need to map with `identity`